maintainers/maintainer-list.nix

```diff
@@ -888,6 +888,12 @@
     githubId = 1296771;
     name = "Anders Riutta";
   };
+  arkivm = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

Reply:
Sure. Should I sort it by `maintainer.mname` or by `maintainer.mname.name`? What would you suggest?

Reply:
The former usually

Reply:
Done. thanks